docs/guides/blocks/cards.md

```diff
@@ -0,0 +1,224 @@
+---
+nav: Cards
+description: Display structured information as rich cards.
+---
+
+# Preview content with rich cards
+
+Pack formulas can return structured data as [objects][data_types_objects], allowing a single formula to return multiple values at once. By default these objects are presented as "mentions", shown as chips in the document that you can hover over to get the full set of information.
```

Comment:
getting a bit confused by the "return multiple values at once" bc that makes me think of arrays, is this just saying an object has multiple properties?

Reply:
Ya, it is hard to phrase. How about this reword?